content/references/rippled-api/public-rippled-methods/transaction-methods/transaction_entry.md

```diff
@@ -27,8 +27,8 @@ An example of the request format:
     "method": "transaction_entry",
     "params": [
         {
-            "tx_hash": "E08D6E9754025BA2534A78707605E0601F03ACE063687A0CA1BDDACFCD1698C7",
-            "ledger_index": 348734
+            "tx_hash": "C53ECF838647FA5A4C780377025FEC7999AB4182590510CA461444B207AB74A9",
+            "ledger_index": 56865245
```

Comment:
I'm guessing these updates are because the previous transaction isn't found on s1 anymore because it's too old? You can still pull it from full history.

There's no harm in doing updates like that, but it's also kind of a Sisyphean effort. The new transaction will rotate out of recent history soon enough, too. The oldest ledger in s1 as of this writing is from May 26 of this year.